fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/size/FHIRDbResourceSize.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.schema.size;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Schema objects related to this resource
```

Comment:
Its the aggregated size for a resource type, not a single resource instance, right?  If so, I think adding this one word will help make that more clear:
```suggestion
 * Schema objects related to this resource type
```

Reply:
done